app/utils/user_ip_address.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = req => {
+  const xHeaderIpAddress = req.headers['x-forwarded-for']
+  let ipAddress
+  if (xHeaderIpAddress !== undefined) {
+    ipAddress = xHeaderIpAddress.split(',')[0]
+  } else {
+    ipAddress = (req.connection && req.connection.remoteAddress) ||
```

Comment:
idk maybe a little comment here to say what this logic does?

Reply:
comments are risky (hard to maintain, code _should_ be self-explanatory) - and I don't think it's required in this case.